### PR TITLE
Add sikulix

### PR DESCRIPTION
**Pull request by offlinehacker**

### CODE REVIEW

File: pkgs/development/tools/sikuli/default.nix [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, makeWrapper, utillinux, jre, jdk, opencv, tesseract, xdotool, wmctrl }:
+
+stdenv.mkDerivation rec {
+  name = "sikulix-${version}";
+  version = "1.1.0.20140219";
```
Comment (offlinehacker) [inline pkgs/development/tools/sikuli/default.nix]: This is from unversioned nightly release, that's why such strange version. Stable release is too old, i was unable to make it work.

**Comment by offlinehacker** [comment]
Don't merge this yet, this package has a few problems, releated to not pointing to deterministic urls. I will try to fix this upstream.

**Comment by 7c6f434c** [comment]
@offlinehacker any success?

**Comment by offlinehacker** [comment]
Well, i will need this some day, but author might release a new version of
sikuli in that time, currently i really don't like the shape sikuli is in,
but again and again can't find a better alternative.

Currently author does not provide versioned builds, and i don't(currently)
have time to setup ci(Travis) for sikuli. Source builds? That's even
harder, don't have time.

Please leave this pr open, so anyone who will want to run sikuli will be
able to find this pr.

**Comment by 7c6f434c** [comment]
Well, if it is useful and mostly works and isn't going to improve soon, maybe it should be simply merged?

I am used to automating things using xmacroplay/xdotool, so I cannot judge the shape of sikuli myself…

**Comment by offlinehacker** [comment]
Looks like author provided some kind of versioned builds, will investigate.

**Comment by 7c6f434c** [comment]
Oh, versioned builds!

**Comment by pSub** [comment]
Any updates?

**Comment by pSub** [comment]
Ping.

**Comment by datakurre** [comment]
For what it is worth, I believe, I managed to build SikuliX on NixOS by using the current versioned releases, reusing this previous work. IDE starts, I can capture screenshots for scripts and execute scripts properly.

https://github.com/datakurre/nix-files/commit/652c4f9656d4ac340143c830b6e165dd7bc4d127

It's still ugly in that on first run it copies libVisionProxy.po into ~/.SikuliX and that probably goes broken once its dependencies are garbage collected, but just removing  ~/.SikuliX seems to be safe.

**Pull request closed without merging**